### DISCUSSION TITLE
Can create game, but nobody can join

### DISCUSSION THREAD

**Discussion by Gretzzy** [Impostor/Impostor#439] [Q&A]
Hello, i have a vserver with Impostor installed and i can create a Game but my friends can't join.

they got the error 
[Imgur link](https://imgur.com/roDAYwV)

You disconnected from the server.

Reliable packet 1 (size=24) was not ack'd after 7523ms (9 resends)

Can anybody help me?

**Comment by miniduikboot** [comment]
Hi there, this issue means that your friends cannot connect to the server. There are a few possible reasons for this:

1. If you're hosting the server at your own house, you may need to portforward Impostor to the outside world. You need to open port 22023 with protocol UDP to the outside world. To check if you did this correctly, you can use https://aucheck.duikbo.at/
2. Did you configure your external IP properly in your config.json file? You can get your external IP on a site like https://icanhazip.com/

This forum isn't as actively monitored as our Discord, if you still need help you may be better off asking there: https://discord.gg/Mk3w6Tb
